ld/testsuite/ld-riscv-elf/relocid.d

```diff
@@ -0,0 +1,25 @@
+#source: relocid.s
```

Comment:
I think it could be nice to have filenames be prefixed with `cv-hwlp-` to show where they're from (and also precise that this test is for ui12/starti)